src/backend/templates/schedule.hbs

```diff
@@ -539,9 +542,18 @@
                     $(sessionName).html(textArray.join(' ') + '...');
                 }
               })
+              roomCount++;
             });
+
             if(width < $(rooms).width()) {
-              $(rooms).width(width);
+              offset = ($(rooms).width() - width)/roomCount;
```

Comment:
Assuming we have a full-width container that will be 100vw ( 100% of the viewport ). 
I doubt why we need anything from frontend, we can structure the table like this.
- Calculate the width the table needs from server => It will be calculated as 
            container width = no. of columns * width  per column
Assuming we always want to give a table a full width.
width per-column = container width / no.of columns ( calculated at server )
This can be sent back to the client to take CSS.

I don't know if you are thinking complex cases for Responsiveness but we are not going into that for now.
Hope this helps :)